modulemd/v2/modulemd-module-stream.c

```diff
@@ -0,0 +1,346 @@
+/*
+ * This file is part of libmodulemd
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Fedora-License-Identifier: MIT
+ * SPDX-2.0-License-Identifier: MIT
+ * SPDX-3.0-License-Identifier: MIT
+ *
+ * This program is free software.
+ * For more information on the license, see COPYING.
+ * For more information on free software, see <https://www.gnu.org/philosophy/free-sw.en.html>.
+ */
+
+#include "modulemd-module-stream.h"
+#include "modulemd-module-stream-v1.h"
+#include "modulemd-module-stream-v2.h"
+#include "private/modulemd-module-stream-private.h"
+#include "private/modulemd-util.h"
+
+typedef struct
+{
+  guint64 mdversion;
```

Comment:
Is there a specific reason you're also storing this if the getter returns the mdversion of the child class?

Reply:
Nope, relic of an earlier thought-process. Removed.